decidim-meetings/lib/decidim/meetings/engine.rb

```diff
@@ -113,6 +113,22 @@ class Engine < ::Rails::Engine
       initializer "decidim_meetings.webpacker.assets_path" do
         Decidim.register_assets_path File.expand_path("app/packs", root)
       end
+
+      initializer "decidim_meetings.notification_settings" do
+        Decidim.notification_settings(:close_meeting_reminder) { |ns| ns.settings_area = :administrators }
+      end
+
+      initializer "decidim_meetings.register_reminders" do
+        Decidim.reminders_registry.register(:close_meeting) do |reminder_registry|
+          reminder_registry.generator_class_name = "Decidim::Meetings::CloseMeetingReminderGenerator"
+          reminder_registry.form_class_name = "Decidim::Meetings::Admin::CloseMeetingReminderForm"
```

Comment:
Here I also believe you don't have to configure the form and command if you are not using them.